kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/json/JSONMarshaller.java

```diff
@@ -306,6 +323,10 @@ public String marshall(Object input, Map<String, Object> parameters) {
             if (parameters.containsKey(MARSHALLER_PARAMETER_STRICT)) {
                 jsonContext.get().setWrap((boolean) parameters.get(MARSHALLER_PARAMETER_STRICT));
```

Comment:
This casting to boolean is failing when receiving a String parameter (true) due to changes for invoking _buildParameters_ instead of _isStrictType_ .
Failing test is _IdentifierJbpmRestIntegrationTest.testBasicJbpmRequest_ 
If always receives a String, it should be:
```suggestion
                jsonContext.get().setWrap(Boolean.parseBoolean((string) parameters.get(MARSHALLER_PARAMETER_STRICT)));
```

Reply:
Done